### PR TITLE
Update SA1600 to show inheritance example.

### PR DESCRIPTION
**Pull request by codajo**

### CODE REVIEW

File: documentation/SA1600.md [hunk 1]
```diff
@@ -44,6 +44,36 @@ public string JoinNames(string firstName, string lastName)
 }
 ```
 
+The next example shows a method that is inherited from a base class:
```
Comment (sharwell) [inline documentation/SA1600.md:47]:
```suggestion
The next example shows a method that inherits documentation from a method in the base class:
```

File: documentation/SA1600.md [hunk 2]
```diff
@@ -44,6 +44,36 @@ public string JoinNames(string firstName, string lastName)
...
+```csharp
+/// <summary>
+/// Example base class.
+/// </summary>
+class Vehicle 
+{
+  /// <summary>
+  /// Accelerates the vehicle.
+  /// </summary>
+  public virtual void accelerate()
```
Comment (sharwell) [inline documentation/SA1600.md:58]:
```suggestion
  public virtual void Accelerate()
```

File: documentation/SA1600.md [hunk 3]
```diff
@@ -44,6 +44,36 @@ public string JoinNames(string firstName, string lastName)
...
+  }
+}
+
+/// <summary>
+/// Example derived class.
+/// </summary>
+class Car : Vehicle
+{
+  /// <inheritdoc/>
+  public override void accelerate()
```
Comment (sharwell) [inline documentation/SA1600.md:70]:
```suggestion
  public override void Accelerate()
```

File: documentation/SA1600.md (hunk 1 above)
Comment (codajo) [reply documentation/SA1600.md:47]: Looks good.

File: documentation/SA1600.md (hunk 2 above)
Comment (codajo) [reply documentation/SA1600.md:58]: Looks good. I could not hide my java background :)

File: documentation/SA1600.md (hunk 3 above)
Comment (codajo) [reply documentation/SA1600.md:70]: Looks good.

**Review by sharwell — APPROVED** [review]

**Comment by codajo** [comment]
Can the change be merged to master or is there another action on my side?

**Comment by sharwell** [comment]
It should automaticalyl merge once the build completes.

**Comment by MartyIX** [comment]
It did not. 🤡

**Pull request merged**